### ISSUE TITLE
Grey Window

### ISSUE THREAD

**Issue by jHoyne** [dfpeterson/GetDuolingoNotes#2]
When the script is run it produces a grey screen in the Tab.
![screenshot 2016-12-30 at 18 15 41](https://cloud.githubusercontent.com/assets/6077549/21574635/069e25cc-cebc-11e6-8f27-44dee9a29b3e.png)

**Comment by dfpeterson** [comment]
I just replicated the problem. I wonder if they changed their code, I'll have to try to debug it.

**Comment by InputNeuron** [comment]
Yeah. I have the same problem now.  I might have some time today or tomorrow to help you debug it..

**Comment by dfpeterson** [comment]
A couple of weeks ago I was able to determine that the issue was that they changed the methodology for storing the notes. You used to be able to go in and get the notes via the main module the same way you can with the words, but they appear to have changed it and I suspect that it got moved to a java function but I was unable to change the function that called the notes. They appear to have changed the way that the website functions again recently and have not been able to look at the code on the new site, but I imagine that the notes are still hidden behind a function. If anyone can determine how to call the notes we can fix this function.
